packages/dx-react-grid-bootstrap3/src/templates/table-tree-checkbox.jsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+
+import { SelectionControl } from './parts/selection-control';
+
+export const TableTreeCheckbox = ({
```

Comment:
A bit strange name. `TreeRowCheckbox`? `TreeTableCheckbox`? And why do we need this 'Tree' part in the name?  Is it really so special?

Reply:
This is internal name. It should be specific enough as it present in a global namespace.